src/blueapi/service/interface.py

```diff
@@ -48,6 +50,19 @@ def worker() -> TaskWorker:
     return worker
 
 
+@cache
+def tiled_inserter():
```

Comment:
Should:

```suggestion
def tiled_inserter() -> TiledClient:
```

...or whatever it's called

Reply:
Now called tiled-write and includes the approriate return type